metricflow/model/data_warehouse_model_validator.py

```diff
@@ -217,6 +217,72 @@ def gen_dimension_tasks(
                 )
         return tasks
 
+    @classmethod
+    def gen_identifier_tasks(
+        cls, model: UserConfiguredModel, sql_client: SqlClient, system_schema: str
+    ) -> List[DataWarehouseValidationTask]:
+        """Generates a list of tasks for validating the identifiers of the model
+
+        The high level tasks returned are "short cut" queries which try to
+        query all the identifiers for a given data source. If that query fails,
+        one or more of the identifiers is incorrectly specified. Thus if the
+        query fails, there are subtasks which query the individual identifiers
+        on the data source to identify which have issues.
+        """
+
+        render_tools = QueryRenderingTools(model=model, system_schema=system_schema)
+
+        tasks: List[DataWarehouseValidationTask] = []
+        for data_source in model.data_sources:
+            if len(data_source.identifiers) > 0:
+                source_node = cls._data_source_node(render_tools=render_tools, data_source=data_source)
+
+                data_source_sub_tasks: List[DataWarehouseValidationTask] = []
+                data_source_specs: List[MeasureSpec] = []
+                for identifier in data_source.identifiers:
+                    spec = IdentifierSpec.from_name(name=identifier.name)
+                    data_source_specs.append(spec)
+                    filter_elements_node = FilterElementsNode(parent_node=source_node, include_specs=[spec])
```

Comment:
You know what I'm going to suggest in here. 😛 

The accessor should be instance_set.spec_set.identifier_specs